src/notebooklm/cli/session.py

```
@@ -47,6 +55,185 @@
+            f"Error: {e}\n\n"
+            "Saved anyway, but you may need to re-run login if these are invalid."
+        )
+    except (ConnectionError, TimeoutError) as e:
+        # Network error - can't verify but cookies might be OK
+        logger.warning(f"Could not verify cookies due to network error: {e}")
+        console.print(
+            "[yellow]Warning: Could not verify cookies (network issue).[/yellow]\n"
+            "Cookies saved but may not be working.\n"
+            "Try running 'notebooklm ask' to test authentication."
+        )
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)